docs/cloud/dev/spryker-cloud-commerce-os/front-end-enablement-service/front-end-enablement-service.md

```diff
@@ -19,7 +19,7 @@ Mobile-first design.
 
 ## Who the users of FeS are?
 
-FeS is designed to be used by devlopers and content managers.
+FeS is designed to be used by developers and content managers.
```

Comment:
FES

Reply:
@alistairSpryker I think "content managers" is not the right user name for now. We could start this docs with developers for now.